reliquery/tests/test_html.py

```diff
@@ -0,0 +1,50 @@
+import os
+import reliquery.relic as relic
+
+
+def test_list_html_file_when_add_html(tmpdir):
+    reliquery_dir = os.path.join(tmpdir, "reliquery")
+    os.makedirs(reliquery_dir)
+
+    rq = relic.Relic(name="element-bucket", relic_type="test")
+
+    rq.add_html(
+        "test html",
+        "/home/welsh/Workspace/projects/reliquery/reliquery/tests/test.html",
```

Comment:
this path will only work on your machine. instead you may want to create your html test file in the tmpdir.

https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture <- here is an example of using tmpdir to write a file. you can write a file that contains html init as part of your test.

Reply:
pointed to relative path if that is fine